client/client.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+const port = 3000;
+
+app.use(express.static(path.join(__dirname, 'src')));
+app.get('/', (req, res) => {
+    res.sendFile(path.join(__dirname, 'src', 'visjs.html'));
+});
+
+app.listen(port, () => {
+    console.log(`Server listening at http://localhost:${port}`);
+});
```

Comment:
Can you add a line in README how to run the client

Reply:
Sure. Will do it